mesa/space.py

```diff
@@ -34,12 +37,44 @@ class Grid(object):
         get_cell_list_contents: Returns the contents of a list of cells ((x,y) tuples)
     '''
 
-    width = None
-    height = None
-    torus = False
-    grid = None
     default_val = lambda s: None
 
+
+    class GridOccupiedIter:
```

Comment:
Yes, renaming it OccupiedCells would be fine.

On Mar 10, 2015, at 11:32 PM, Jackie Kazil <notifications@github.com> wrote:

> In mesa/space.py:
> 
> >      default_val = lambda s: None
> >  
> > +
> > +    class GridOccupiedIter:
> I like where you are going with that, but since this is a subclass of a class called Grid, that would ... 
> Grid.GridOccupiedCells
> 
> Maybe -- Grid.OccupiedCells? 
> How about ... 
> class OccupiedCells
> Yes? No? 
> (If you agree -- update. if not, then let me know you ur thoughts.)
> 
> —
> Reply to this email directly or view it on GitHub.
>